eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -63,15 +63,25 @@ initiateHardForkWithLessThanMinimalCommitteeSize ::
   SpecWith (ImpTestState era)
 initiateHardForkWithLessThanMinimalCommitteeSize =
   it "Hard Fork can still be initiated with less than minimal committee size" $ do
-    (hotC :| _) <- registerInitialCommittee
+    (_hotC1 :| [hotC2]) <- registerInitialCommittee
+    (spoK1, _, _) <- setupPoolWithStake $ Coin 1_000
     passEpoch
     modifyPParams $ ppCommitteeMinSizeL .~ 2
+    committee <- Set.toList <$> getCommitteeMembers
+    resignCommitteeColdKey (head committee) SNothing
```

Comment:
I suspect this will result in a CI failure, since usage of `head` will not emit a warning.

I'd also suggest picking one at random anyways
```suggestion
    committeeMembers <- getCommitteeMembers
    committeeMember <- elements $ Set.toList committeeMember
    resignCommitteeColdKey committee SNothing
```